gunicorn/glogging.py

```diff
@@ -204,9 +204,9 @@ def setup(self, cfg):
             for stream in sys.stdout, sys.stderr:
                 stream.flush()
 
-            self.logfile = open(cfg.errorlog, 'a+')
-            os.dup2(self.logfile.fileno(), sys.stdout.fileno())
-            os.dup2(self.logfile.fileno(), sys.stderr.fileno())
+            with open(cfg.errorlog, 'a+') as self.logfile:
+                os.dup2(self.logfile.fileno(), sys.stdout.fileno())
+                os.dup2(self.logfile.fileno(), sys.stderr.fileno())
```

Comment:
The problem with this is that, it will set the `self.logfile` to a closed file (since after this context is finalized, it will close the file). So there is no point of setting it to an attribute, since none of the users of that attribute will able to benefit from it.

Reply:
i don't see the point of using a with statement there.  please keep it that way.